packages/framework/src/Model/Feed/FeedExport.php

```diff
@@ -29,7 +31,17 @@ class FeedExport
     /**
      * @var \League\Flysystem\FilesystemInterface
      */
-    protected $filesystem;
+    private $filesystem;
```

Comment:
was this change somehow connected to the implementation of `private $localFileSystem` in `packages/framework/src/Model/Feed/FeedExportFactory.php` or what is the reason for changing the scope to private  ?